File: bcw_2023summer_keepr/Models/Keep.cs
namespace bcw_2023summer_keepr.Models
{
    public class Keep : RepoItem<int>
    {
        public string CreatorId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string Img { get; set; }
        public int Views { get; set; }
        public int Kept { get; set; }
        public Profile Creator { get; set; }
    }

    public class KeepRelationship : Keep
    {
        public int VaultKeepId { get; set; }
    }
}

File: bcw_2023summer_keepr.client/src/models/Keep.js
export class Keep {
	constructor(data) {
		this.id = data.id;
		this.createdAt = data.createdAt;
		this.updatedAt = data.updatedAt;
		this.creatorId = data.creatorId;
		this.name = data.name;
		this.description = data.description;
		this.img = data.img;
		this.views = data.views;
		this.kept = data.kept;
		this.creator = data.creator;
		this.vaultkeepid = data.vaultkeepid;
	}
}